lib/Jit/LLILCJit.cpp

```diff
@@ -641,3 +641,12 @@ void ObjectLoadListener::getRelocationTypeAndAddend(uint8_t *FixupAddress,
     llvm_unreachable("Unknown reloc type.");
   }
 }
+
+unsigned LLILCJit::getMaxIntrinsicSIMDVectorLength(DWORD CpuCompileFlags) {
+  return GenIR::getPreferIntrinsicSIMDVectorLength();
+}
+
+bool LLILCJitContext::isAltJit() {
+  JitOptions JitOptions(*this);
+  return JitOptions.IsAltJit;
```

Comment:
Options are available to the reader, they would be used for both the JIT and AOT path - anything that used the MSIL reader path.  The JIT options are specific to the JIT client.  When we have AOT this would include AOT specific options (and different queries as well) distinct from JIT. That's why we have the separation.  If your option is used in the reader then it needs to be exposed via the base options.  It looks like you're using this in readerir.cpp, so my question is why do you need to know that this is the AltJit?  What would this look like if you were supporting SIMD for AOT path?